minor server configuration changes

notwithering opened the pull request:
feel free to coauthor

buraktrker reviewed (changes requested):
Thanks for request,

I changed project structure and it is very different now. I split config and server(daemon with new name).

With this structure serverconfig usable in daemon and server cli app.


For your changes can u look at: [serverconfig.go](https://github.com/burakturkerdev/ftgo/blob/main/src/server/lib/serverconfig.go
)
Thanks again :)

notwithering commented:
> Thanks for request,
> 
> I changed project structure and it is very different now. I split config and server(daemon with new name).
> 
> With this structure serverconfig usable in daemon and server cli app.
> 
> For your changes can u look at: [serverconfig.go](https://github.com/burakturkerdev/ftgo/blob/main/src/server/lib/serverconfig.go) Thanks again :)

Yes, of course, I will make the necessary changes right away.

notwithering committed:
PR #1 update
